release-content/release-notes/headless-widgets.md

```diff
@@ -84,9 +84,10 @@ interacting with that widget. Externalizing the state avoids the need for two-wa
 instead allows simpler one-way data binding that aligns well with the traditional "Model / View /
 Controller" (MVC) design pattern.
 
-That being said, the choice of internal or external state management is up to you: if the widget
-has an `on_change` callback that is not `None`, then the callback is used. If the callback
-is `None`, however, the widget will update its own state. (This is similar to how React.js does it.)
+That being said, the choice of internal or external state management is up to you: if the widget has
```

Comment:
Non-blocking: I would love a link to internal vs external state management here before 0.17 ships.

Reply:
Revised the docs and added a link.